packages/sanity/src/core/config/types.ts

```diff
@@ -291,10 +291,15 @@ export interface DocumentPluginOptions {
    */
   newDocumentOptions?: NewDocumentOptionsResolver
 
-  /** @internal */
+  /** @deprecated - use `comments`instead */
```

Comment:
Nitpick: should we align this deprecation message with the one for Tasks – or vice versa?

```diff
- /** @deprecated - use `comments` instead */
+ /** @deprecated  Use `comments` instead */
```

Reply:
I prefer ```/** @deprecated  Use `comments` instead */``` since the dash is added by JSDoc. 

<img width="275" alt="Screenshot 2024-04-24 at 17 45 58" src="https://github.com/sanity-io/sanity/assets/15094168/d5108fc5-7331-4c72-8dfb-72f39961237b">